spec/rubocop/cop/rails/exit_spec.rb

```diff
@@ -0,0 +1,32 @@
+# encoding: utf-8
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe RuboCop::Cop::Rails::Exit, :config do
+  subject(:cop) { described_class.new }
+
+  it 'registers an offense for an exit(0) call' do
+    inspect_source(cop,
+                   'exit(0)')
+    expect(cop.offenses.size).to eq(1)
+    expect(cop.messages)
```

Comment:
@bbatsov i like to check to make sure that the offense is the one we expect, and not a different offense. thoughts?